metagpt/prompts/tutorial_assistant.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# _*_ coding: utf-8 _*_
+"""
+@Time    : 2023/9/4 15:40:40
+@Author  : Stitch-z
+@File    : tutorial_assistant.py
+@Describe : Tutorial Assistant's prompt templates.
+"""
+
+COMMON_PROMPT = """
+You are now a seasoned technical professional in the field of the internet. 
+We need you to write a technical tutorial with the topic "{topic}".
+"""
+
+DIRECTORY_PROMPT = COMMON_PROMPT + """
+Please provide the specific table of contents for this tutorial, strictly following the following requirements:
+1. The output must be strictly in the specified language, {language}.
+2. Answer in the dictionary format like {{"title": "xxx", "directory": [{{"dir 1": ["sub dir 1", "sub dir 2"]}}]}}.
```

Comment:
The reason for using '{{}}' instead of '{}' in this template string is to avoid conflicts with '{}' in the template string, where "{{" indicates that it is text content "{". Otherwise, DIRECTORY_ PROMPT. format (topic="xx", language="xx") may have parsing failure issues.

Reply:
`'{lang} {{"title": "xxx"}} {topic}'.format(lang="en", topic="test", title="test2")
'en {"title": "xxx"} test'`. Do you happened with the failure case?

Reply:
![image](https://github.com/geekan/MetaGPT/assets/56247966/2d87e469-5c3f-4a1a-9261-938fad6714f4)
Yes, it will throw a key error.